server/wfone-notification-push-api/wfone-notification-push-api-rest-endpoints/src/main/resources/firebase-config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
We can remove the client ID and URL info and move that to configurations/secrets, that would be ideal. Rather not have these checked in

Reply:
Thanks I've done so now